### PR TITLE
Office page template

### LINKED ISSUE #5602: Build office page template
**What we're after:**
Develop new office page templates per designs agreed to by FEC leadership. [Page layouts are in Mural](https://app.mural.co/t/fec4461/m/fec4461/1675268367751/9ddad9551d00a89a468b900e565729c5f1d13b95?sender=jculmer4627). 

### Related issues
https://github.com/fecgov/fec-cms/issues/4376

### Completion criteria
- [ ] Office page template built according to [annotated page layout in Mural](https://app.mural.co/t/fec4461/m/fec4461/1675268367751/9ddad9551d00a89a468b900e565729c5f1d13b95?sender=jculmer4627)

### PR DESCRIPTION
**Pull request by johnnyporkchops**
## Summary
Office page template. [Mural board of mockups](https://app.mural.co/t/fec4461/m/fec4461/1675268367751/9ddad9551d00a89a468b900e565729c5f1d13b95?sender=0bb5f432-316e-4530-91a5-0470dd7a1e46) for pages that will need to be built with this template
- Resolves #5602

Currently deployed to dev space for testing - Mockups: 
https://dev.fec.gov/about/test-office/
https://dev.fec.gov/admin/pages/12755/view_draft/

### Required reviewers

One UX , one frontend

## Impacted areas of the application

MODIFIED: fec/static/scss/base.scss
MODIFIED: fec/static/scss/components/_contact-items.scss
MODIFIED: fec/home/blocks.py
MODIFIED: fec/home/models.py
MODIFIED: fec/home/templates/blocks/contact-info.html
NEW: fec/static/scss/components/_fec-offices.scss
NEW: fec/home/templates/blocks/simple-document.html
NEW: fec/home/templates/home/office_page.html
NEW: fec/home/migrations/0125_officepage.py

## Screenshots

Screen Shot 2023-03-15 at 4 05 38 PM

## How to test
- [x] remove dev deploy task before merge
- `npm run build-sass`
-  Go to Wagtaill admin
- Test new OfficePage template
See examples here:
[Mural board of mockups](https://app.mural.co/t/fec4461/m/fec4461/1675268367751/9ddad9551d00a89a468b900e565729c5f1d13b95?sender=0bb5f432-316e-4530-91a5-0470dd7a1e46)

- Test pages are also created on Dev, such as: 
    -  https://dev.fec.gov/about/test-office/
    -  https://dev.fec.gov/admin/pages/12755/view_draft/
    
**Note:** The only `subpage_types` dict it was added to is `AboutLandingPage`, but it works as a child of the more generic page  types like ResourcePage and CustomPage because they have no  dict restricting  allowable subpage_types`.  If we need it as a child of any of the other other pages with restrictions, we need to add it to their lists.

### CODE REVIEW

**Comment by JonellaCulmer** [comment]
@johnnyporkchops Everything looks great in dev. Just noted two things:
1. When linking to other pages like we will in the mockup below, will we able to use anchor links to those specific sections? Each office needs to be/have it's own section ID, right?
Screen Shot 2023-03-10 at 10 49 49 AM

2. For footnotes, there isn't a clear place to enter a footnote at the bottom of the page, so I'm just using the extra information section at the bottom of the office section. Easy enough to do it that way but I was wondering if that was your intention?

**Comment by johnnyporkchops** [comment]
> 1. When linking to other pages like we will in the mockup below, will we able to use anchor links to those specific sections? Each office needs to be/have it's own section ID, right?

@JonellaCulmer. Correct.  I'll  add a class or id to the H2 (or above the H2) that matches he office title replacing spaces with underscores or dashes. So it would be something like `id='office_of_the_commission_secretary'`. Then you could anchor link to is with an external link in the text editor.

> 2. For footnotes, there isn't a clear place to enter a footnote at the bottom of the page, so I'm just using the extra information section at the bottom of the office section. Easy enough to do it that way but I was wondering if that was your intention?

Yes,  thats the idea. Since it works that way, we don't need to create another field specifically for that

**Comment by johnnyporkchops** [comment]
@JonellaCulmer foe anchoring to an office section, I will add  a separate  anchor item  above the H2-title for the office section instead of on the H2 to handle cases where we leave the H2-title off. 
(There is also the anchor option in Rich Text fields which might come in handy in certain situations, although it has limited capability )

File: tasks.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def _detect_space(repo, branch=None, yes=False):
 DEPLOY_RULES = (
     ('prod', _detect_prod),
     ('stage', lambda _, branch: branch.startswith('release')),
-    ('dev', lambda _, branch: branch == 'develop'),
+    ('dev', lambda _, branch: branch == 'feature/5602-office-page-template'),
```
Comment (patphongs) [inline tasks.py:77]: Please update back to develop branch

**Review by patphongs — CHANGES_REQUESTED** [review]
Changes look great and works as expected @johnnyporkchops, just a small edit to put the develop branch back for the deploy hook. Then I will fully approve.

**Review by JonellaCulmer — APPROVED** [review]
Everything looks great! Thanks, @johnnyporkchops

**Review by patphongs — APPROVED** [review]
👍 Thanks @johnnyporkchops !

**Pull request merged**